src/modules/NgGrid.module.ts

```diff
@@ -0,0 +1,13 @@
+import { NgModule, ModuleWithProviders } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { BrowserModule } from '@angular/platform-browser';
+import { FormsModule } from '@angular/forms';
+import { NgGrid, NgGridItem, NgGridItemConfig, NgGridItemEvent, NgGridPlaceholder } from '../main';
+
+@NgModule({
+  imports:          [ BrowserModule, CommonModule, FormsModule ],
```

Comment:
Do we need to import all of these? The forms module, for example, isn't used by the library